package-template.sile-dev-1.rockspec

```diff
@@ -0,0 +1,28 @@
+package = "package-template.sile"
+version = "dev-1"
+
+description = {
+  summary = "A template for SILE packages...",
+  detailed = [[]],
+  homepage = "", -- e.g. its GitHub repo
+  maintainer = "",
+  license = "MIT"
+}
+
+source = {
+  url = "", -- a clonable repository link
+  -- tag = ""
+}
+
+dependencies = {}
```

Comment:
I would show an example, and the minimal one might be:

```
dependencies = {
   "lua >= 5.1",
}
```

(Assuming this repository also provides an adequate `.luacheckrc` and a GitHub action, this would be neat.)